lib/fetch/request.js

```diff
@@ -823,7 +823,7 @@ function makeRequest (init) {
       ? new HeadersList(init.headersList)
       : new HeadersList()
   }
-  request.url = request.urlList[0]
+  request.url = request.urlList[request.urlList.length - 1]
```

Comment:
Bug?

I assume that urlList lenght - 1 has to be used?

Reply:
request's URL is the first one, not the last